src/main/java/racingcar/controller/RaceController.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.controller;
+
+import racingcar.model.*;
+import racingcar.view.validator.InputValidator;
+import racingcar.view.InputView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RaceController {
+    private final InputView inputView = new InputView();
+    private final InputValidator inputValidator = new InputValidator();
+
+    public void run() {
+        List<String> carNames = inputView.getCarNames();
+        inputValidator.validateAvailableCarNames(carNames);
+        int attemptNumber = inputView.getAttemptNumber();
+
+        Cars cars = new Cars(generateCarListByCarNames(carNames));
+        Race race = new Race(cars, attemptNumber);
+        race.runRace();
+    }
+
+    private List<Car> generateCarListByCarNames(List<String> carNames) {
+        return carNames.stream().map(carName -> new Car(carName, new RandomNumberGenerator())).collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    private List<Car> generateCarListByCarNames(List<String> carNames) {
        return carNames.stream()
            .map(carName -> new Car(carName, new RandomNumberGenerator()))
            .collect(Collectors.toList());
    }
```
한 줄에 점 하나씩!
또한, `List<Car>`를 만들어서 `Cars`를 생성하기보다는 `Cars`가 내부적으로 생성하도록 책임을 넘겨줄 수 있지 않을까요?
도비가 생각하는 `Cars`의 책임은 무엇인가요? 🤔

Reply:
일급컬렉션으로서 list로 품고 있는 객체들에 대한 기본적인 데이터 처리 및 가공에 대한 책임을 가져야된다고 생각하고 있습니다!
말씀해주신 부분 생각해보니 지니고 있을 객체를 직접 생성하는 역할 또한 처리 및 가공 역할이라고 할 수 있을 것 같습니다!!